src/prpy/serialization.py

```diff
@@ -16,7 +16,9 @@ def serialize(obj):
 
     NoneType = type(None)
 
-    if isinstance(obj, (int, float, basestring, NoneType)):
+    if isinstance(obj, numpy.float64):
```

Comment:
Replace `numpy.float65` with `numpy.floating`, which is the base class for all NumPy floating point types (`float`, `float16`, `float32`, `float64`, and `float128`. You may want to add similar logic to convert `numpy.signedinteger` and `numpy.unsignedinteger` to `int`s.